src/sentry/issues/related/same_root_cause.py

```diff
@@ -3,29 +3,26 @@
 # The first case this module handles is environmental failures.
 #
 # Refer to README in module for more details.
-from typing import Any
-
 from sentry.models.group import Group
 from sentry.utils.query import RangeQuerySetWrapper
 
 
-def same_root_cause_analysis(group: Group) -> list[Group]:
+def same_root_cause_analysis(group: Group) -> list[int]:
```

Comment:
The endpoint is going to return the group IDs. This also makes typing a bit simpler in the module above.